lib/flash_flow/issue_tracker/pivotal.rb

```diff
@@ -74,17 +85,44 @@ def finish(story_id)
       def comment(story_id)
         story = get_story(story_id)
         if story
-          note_prefix = 'Shipped to production on'
-          unless already_has_comment?(story, Regexp.new("^#{note_prefix}"))
-            story.notes.create(:text => Time.now.strftime("#{note_prefix} %m/%d/%Y at %H:%M"))
+          unless has_shipped_text?(story)
+            story.notes.create(:text => Time.now.strftime(note_time_format))
           end
         end
       end
 
+      def note_prefix
+        'Shipped to production on'
+      end
+
+      def note_time_format
+        "#{note_prefix} %m/%d/%Y at %H:%M"
+      end
+
+      def format_release_data(story_id, story_name, shipped_text)
+        {id: story_id, title: story_name, time: Time.strptime(shipped_text, note_time_format)}
+      end
+
       def shipped?(branch)
         branch.sha && @git.master_branch_contains?(branch.sha)
       end
 
+      def done_and_current_stories
+        [@project.iteration(:done).last(2).map(&:stories) + @project.iteration(:current).stories].flatten
+      end
```

Comment:
Yes I noticed that. So the farthest back it will look is three weeks. Thank you for the heads up, I agree with your decision to limit the search, it would take way too long otherwise.